modules/common/src/main/scala/surge/internal/streams/StreamManager.scala

```diff
@@ -207,7 +221,7 @@ class KafkaStreamManagerActor[Key, Value](
     case _            => stash()
   }
 
-  private def startConsumer(): Unit = {
+  private def startConsumer(tracer: Tracer): Unit = {
```

Comment:
Any reason in particular to pass the tracer to this function rather than leverage the one passed to the constructor? Looks like that's the one passed along anyways from the `Receive` function

Reply:
@jeffboutotte I think you are correct, I'll fix it.